client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -260,6 +260,11 @@ $.entwine('ss', function($){
      * Suppress submission unless it is handled through ajaxSubmit().
      */
     onsubmit: function(e, button) {
+      // If the submit button a child of on an element with a .bypass-entwine class on it then
+      // skip handling. Usually the element in question will be used by react e.g. FormBuilder
+      if ($(e.target.activeElement).closest('.bypass-entwine').length) {
```

Comment:
```suggestion
      if ($(e.target.activeElement).closest('.bypass-entwine-submission').length) {
```
Make the css class more specific - even this may not be specific enough but at least it gives a hint about what is actually being bypassed.

Reply:
Why are we using `closest`?
IIRC this will check all elements up and down the hierarchy in the DOM - feels a bit too broad a check?

Reply:
Updated

closest() only goes up - `For each element in the set, get the first element that matches the selector by testing the element itself and traversing up through its ancestors in the DOM tree.` https://api.jquery.com/closest/